src/FSharp.SystemTextJson/All.fs

```diff
@@ -13,13 +13,13 @@ type JsonFSharpConverter(fsOptions: JsonFSharpOptions) =
     static member internal CreateConverter(typeToConvert, options, fsOptions) =
         match TypeCache.getKind typeToConvert with
         | TypeCache.TypeKind.List ->
-            JsonListConverter.CreateConverter(typeToConvert)
+            JsonListConverter.CreateConverter(typeToConvert, fsOptions)
         | TypeCache.TypeKind.Set ->
-            JsonSetConverter.CreateConverter(typeToConvert)
+            JsonSetConverter.CreateConverter(typeToConvert, fsOptions)
         | TypeCache.TypeKind.Map ->
             JsonMapConverter.CreateConverter(typeToConvert)
```

Comment:
This should probably be null-checked too, but maybe that can be done in another PR